vllm/v1/executor/multiproc_executor.py

```diff
@@ -34,10 +35,25 @@
 class MultiprocExecutor(Executor):
 
     def __init__(self, vllm_config: VllmConfig) -> None:
-        # Call self.shutdown at exit to clean up
```

Comment:
NOTE: removed this because this creates a circular reference + `EngineCore` calls `executor.shutdown()` at its exit.